EmerisHQ/demeris
# #1256 Fix restaking in CRO

## Description

There was an error when attempting to unstake/restake CRO. In my research it appears to be related to the validator address being incorrect. The validator address is built in the frontend by encoding the prefix and address with bech32. But the prefix returned by the backend in this endpoint `https://api.emeris.com/v1/chain/crypto-org` in the property chain.node_info.bech32_config.val_address was returning the wrong value. Alex Cheng updated it to return the correct value now. But this introduced another bug which is being addressed in a backend ticket (see below).

Fixes: # 1256

## Testing

Steps to reproduce:

-     Switch Validator in CRO
-     Select new validator to restake to
-     Staked amount should now be visible


* After switching succesfully I'm curently stuck in processing the transaction 
![image](https://user-images.githubusercontent.com/60057887/158546650-21f4a383-8af3-4630-902d-9c183c470554.png)

And the following API calls fail:
![image](https://user-images.githubusercontent.com/60057887/158547203-10783e0d-9d5f-477e-b293-2b46f9dfdddb.png)

https://api.emeris.com/v1/tx/ticket/cosmos-hub/F239D32CF78D5EBA86358D346185AB43A7DC814A76A08161B360C862984E0915

---

## Additional Details (Optional)

A backend ticket to remove the non-zero whitespace has been created [here](https://app.zenhub.com/workspaces/emeris-back-end-619cd009cc778000112d6259/issues/emerishq/demeris-backend/574) 

I have added a TODO comment in the src/utils/basic.ts to then remove the replace which will become redundant.
